lib/l10n-en_GB.js

```diff
@@ -40,13 +40,13 @@ exports.messages = {
 ,   "headers.dl.this-link":                 "Link href and text differ for This Version."
 ,   "headers.dl.this-date":                 "Mismatch between document date and This Version link."
 ,   "headers.dl.no-date":                   "Cannot find document date."
-,   "headers.dl.this-syntax":               "Wrong syntax for This Version link. Use a 'https' link."
+,   "headers.dl.this-syntax":               "Wrong syntax for This Version link."
```

Comment:
Having a URL that points outside the https://www.w3.org/TR/ space isn't the "wrong syntax", it's a disallowed URL.